ext/rugged/rugged_commit.c

```diff
@@ -107,9 +105,7 @@ static VALUE rb_git_commit_author_GET(VALUE self)
 	git_commit *commit;
 	Data_Get_Struct(self, git_commit, commit);
 
-	return rugged_signature_new(
-		git_commit_author(commit),
-		git_commit_message_encoding(commit));
+	return rugged_signature_new(git_commit_author(commit));
```

Comment:
This is its own can of worms, but for some use-cases we have assumed that the encoding of the signature is the same as of the commit message. This sometimes even holds true, and this is a place where we kinda have a good idea of what the encoding is likely to be, rather than no idea like in the other cases.